modules/installation-aws-config-yaml-customizations.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/ipi/installing-aws-customizations.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="installation-aws-config-yaml-customizations_{context}"]
+= Sample customized install-config.yaml file for AWS
+
+You can customize the installation configuration file (`install-config.yaml`) to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your `install-config.yaml` file by using the installation program and modify it.
+For a full list and description of all installation configuration parameters, see the page titled _Installation configuration parameters_.
```

Comment:
This is a little tricky. Looking at the guidance in [Lead-in link sentences](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences), I think you want this:
```suggestion
For a full list and description of all installation configuration parameters, see _Installation configuration parameters for AWS_.
```
though there is not an example exactly like that